README.md

```diff
@@ -42,9 +42,11 @@ As a starting point for your own project take a look at https://github.com/ether
 
 ##### Running from command line:
 ```
-> git clone https://github.com/ethereum/ethereumj
-> cd ethereumj
-> ./gradlew run [-PmainClass=<sample class>]
+git clone https://github.com/ethereum/ethereumj
+cd ethereumj
+vim ethereumj-core/src/main/resources/ethereumj.conf # adjust ethereumj.conf to your needs
+./gradlew clean shadowJar
+java -jar ethereumj-core/build/libs/ethereumj-core-*-all.jar
 ```
```

Comment:
I’ve addressed your remarks. As I said, this was the part I missed most in the entire `README`. I guess many other users might look for the same solution. Also I’d suggest to provide those executable JARs with every release as this is the only part users care about.